README_zh_CN.md

```diff
@@ -0,0 +1,197 @@
+# 适用于Amazon S3兼容云存储的Minio JavaScript Library [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+[![NPM](https://nodei.co/npm/minio.png)](https://nodei.co/npm/minio/)
+
+Minio JavaScript Client SDK提供简单的API来访问任何Amazon S3兼容的对象存储服务。
+
+本快速入门指南将向您展示如何安装客户端SDK并执行示例JavaScript程序。有关API和示例的完整列表，请参阅[JavaScript客户端API参考](https://docs.minio.io/docs/javascript-client-api-reference)文档。
+
+本文假设你已经安装了[nodejs](http://nodejs.org/) 。
+
+## 使用NPM下载
+
+```sh
+npm install --save minio
+```
+
+## 下载源码安装
```

Comment:
下载并安装源码

Reply:
Acturally I'm not.The code explains itself.Maybe I should change the comment back to English,for consistency.

Reply:
@dingjs SGTM, as long as it's consistent.

@deekoder May be you can comment on if Minio as a whole project would like comments in the code to be translated or remain in English? This can be set as president for future translation work.

Reply:
comments in code can be in english